flow/designs/asap7/mock-array/power.tcl

```diff
@@ -0,0 +1,18 @@
+foreach libFile $::env(LIB_FILES) {
+read_liberty $libFile
+}
+
+read_verilog results/asap7/mock-array_Element/base/6_final.v
+read_verilog $::env(RESULTS_DIR)/6_final.v
+read_verilog designs/src/mock-array/post/empty.v
+
+link_design MockArray
+
+read_sdc $::env(RESULTS_DIR)/6_final.sdc
+read_spef $::env(RESULTS_DIR)/6_final.spef
+# FIXME lots of warnings
+#read_spef results/asap7/mock-array_Element/base/6_final.spef
```

Comment:
Parasitic networks (DSPEF) can be annotated on hierarchical blocks using the -path argument to specify the
instance path to the block. Parasitic networks in the higher level netlist are stitched together at the hierarchical
pins of the blocks.

Reply:
@maliberty I will fix this, but can we merge this PR as a shared baseline so that the various concerns in this PR can be fixed in separate PRs from probably multiple contributors?